components/Footer.tsx

```
@@ -1,10 +1,10 @@
-import { Icons } from "@team-numberone/daepiro-design-system";
+import { IconWrapper } from "./icons/IconWrapper";
 
 export function Footer() {
 	return (
-		<footer className="mt-auto flex justify-center items-center py-[20px] text-caption text-gray-200 gap-[6px]">
-			<Icons.Warning size={"sm"} />
-			<div>이 서비스는 ‘실제 119 연결’이 되지 않습니다.</div>
+		<footer className="flex-8 flex justify-center items-center text-caption text-gray-200 gap-[6px] min-h-[60px]">
+			<IconWrapper name="Warning" size={"sm"} />
+			<div>이 서비스는 '실제 119 연결'이 되지 않습니다.</div>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)